debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouter.java

```diff
@@ -165,7 +165,7 @@ else if (onlyHeadersInOutputMessage) {
         }
 
         R newRecord = r.newRecord(
-                eventStruct.getString(routeByField).toLowerCase(),
```

Comment:
@renatomefi, do you happen to remember why you opted for lower-casing the topic name in the outboux router here? The suggested change to remove it looks reasonable to me, but I wanted to double-check with you whether there's some special intention behind it. Thanks!

Reply:
Hey Gunnar, I don't remember exactly! But as kafka is case sensitive I think it's a fair change.

Reply:
Ok, thanks for the quick confirmation. Will merge.